assets/dev/js/editor/document/repeater/commands/select.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Command used to select current working repeater item.
+ */
+export class Select extends $e.modules.editor.document.CommandHistoryBase {
+	static restore( historyItem, isRedo ) {
+		const container = historyItem.get( 'container' ),
+			data = historyItem.get( 'data' );
+
+		$e.run( 'document/repeater/select', {
+			container,
+			index: isRedo ? data.current : data.prev,
+		} );
+	}
+
+	validateArgs( args ) {
+		this.requireContainer( args );
+		this.requireArgumentType( 'index', 'number', args );
+
+		// When there multiple containers, then its not supported.
+		if ( args.containers ) {
```

Comment:
Why not?  We want to be prepared for multi edit.

Reply:
How we can pass containers of two different tabs in the same time to select command?